### DISCUSSION TITLE
log format

### DISCUSSION THREAD

**Discussion by purushothaman-ottosero** [dbeaver/dbeaver#13642] [Q&A]
Dbeaver adds `/* ApplicationName=DBeaver 21.1.5 - SQLEditor <Script-4.sql> */`, also there is no database name in the audit log.
But with mysql client the log is proper. I am using `MariaDB audit plugin` to capture the audit logs
Normal expected format while using mysql client

`20210824 05:49:46,ip-10-20-3-182,admin,183.xx.xx.xxx,8917,1590713,QUERY,test_env_db,'create table stocklist100 (id int, name varchar(20))',0`

but with dbeaver client the log looks like
`20210824 05:34:47,ip-10-20-3-182,admin,183.xx.xxx.xxx,8912,1590009,QUERY,,'/* ApplicationName=DBeaver 21.1.5 - SQLEditor <Script-4.sql> */ alter table final add column ksentiwnr varchar(1)',0`

Is there way to make the log look like mysql client while using dbeaver?

**Comment by LonwoLonwo** [comment]
Hello @purushothaman-ottosero 
What kind of logs are you talking about?
The screen will be very helpful

**Comment by purushothaman-ottosero** [comment]
I am talking about audit logs for mysql database, a sample log obtained after using dbeaver as client looks like this
![image](https://user-images.githubusercontent.com/77261085/130616928-1b932d8f-9295-431b-9411-e8aa70572b11.png)
with mysql client it looks like this.
![image](https://user-images.githubusercontent.com/77261085/130617323-95659a78-6e6a-4018-a518-4d62ac068209.png)
As you can see there is no database name(one after the `QUERY` in the log) obtained in the log with dbeaver, but with mysql it is available.

**Comment by LonwoLonwo** [comment]
Where can I found "audit logs for mysql database"?

**Reply by purushothaman-ottosero** [reply]
Follow up: I tried connecting with mariadb connector instead of mysql connector in dbeaver and the isssue seems to be resolved, i am getting the database name in the log.

I am using `MariaDB Audit plugin`. It is a plugin for mysql which can be used to capture audit logs. I am using AWS RDS along with the plugin in my current environment setup. You can read about it [here](https://mariadb.com/kb/en/mariadb-audit-plugin/) if you are interested :-). Thanks
